bin/resources/local.rb

```diff
@@ -175,7 +182,6 @@ def start_local_server(project_directory, mongo_directory, ruby_path, worker_num
 
   state_file = ::File.join(project_directory, cluster_name + '_configuration.json')
   receipt_file = ::File.join(project_directory, cluster_name + '_configuration.receipt')
```

Comment:
The .receipt shouldn't be used at present, but I'm not deleting for a little bit (just in case whatever.) server.pid is a rails thing and I don't think we can get rid of it.